javascript/javascript1/week2/jsweek2hw.js

```diff
@@ -0,0 +1,155 @@
+//Flight booking full name function
+function getFullName(firstname,surname)
+{
+ return firstname+' '+surname;   
+}
+const fullname = getFullName("Bhuvaneshwari", "Periyasundaram"); 
+console.log(fullname);
+const fullName1 = getFullName('Krithikaa','Raja');
+const fullName2 = getFullName('Lavanya','Sri');
+console.log(fullName1);
+console.log(fullName2);
+
+
+//formal fullname
+function getFullName(firstname,surname,useFormalName)
+{
+ if(useFormalName)
+ {
+     return 'Lord'+' '+firstname+' '+surname;
+ }
+ else
+ {
+     return firstname+' '+surname;
+ }
+}
+const fullName3 = getFullName('Bhuvaneshwari','Periyasundaram',true);
+const fullName4 = getFullName('Sanju','sri');
+console.log(fullName3);
+console.log(fullName4);
+
+//Event application
+const week = ['sunday','monday','tuesday','wednesday','thursday','friday','saturday','sunday'];
```

Comment:
you have added sunday two times
```suggestion
const week = ['sunday','monday','tuesday','wednesday','thursday','friday','saturday'];
```

Reply:
Removed the extra array input Sunday